crates/ark/src/lsp/symbols.rs

```diff
@@ -424,8 +424,8 @@ fn collect_assignment(
     // Otherwise, collect as generic object
     let name = contents.node_slice(&lhs)?.to_string();
 
-    let start = convert_point_to_position(contents, lhs.start_position());
-    let end = convert_point_to_position(contents, lhs.end_position());
+    let start = convert_point_to_position(contents, node.start_position());
+    let end = convert_point_to_position(contents, node.end_position());
 
     // Now recurse into RHS
```

Comment:
I don't have super deep intuition about how this should work, but from what I can tell:

- If you have an assigned multi line function call, you get a breadcrumb
- If you have a free floating multi line function call, you do not

Is that expected / consistent? It _feels_ like maybe we should get a breadcrumb in both cases? Not totally sure. Probably worth comparing to other languages

https://github.com/user-attachments/assets/ab040749-a3c9-488f-91a8-e286b5c84e37